libs/plugin/src/executors/sync-repo/executor.spec.ts

```diff
@@ -0,0 +1,14 @@
+import type { SyncRepoExecutorSchema } from './schema'
+import executor from './executor'
+
+const options: SyncRepoExecutorSchema = {
+  repoUrl: '',
+  localPath: '',
+}
+
+describe.skip('SyncRepo Executor', () => {
+  it('can run', async () => {
+    const output = await executor(options)
+    expect(output.success).toBe(true)
```

Comment:
does this actually test anything? i would almost expect it to fail on empty strings...

Reply:
It's the default, and skipped rn. Any idea what to test on something that fetch a repo?

Reply:
we could fetch a repo that will never get removed from github. I recommend https://github.com/chrislgarry/Apollo-11/